<launch>
				<!--  ************** Odometry ***************  -->
	<arg name="gui" default="True" />
	<param name="use_gui" value="$(arg gui)"/>
	<param name="robot_description" command="cat $(find robot1_chassis_description)/urdf/nox.urdf" />
	
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
	
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	
	<node name="serial_node" pkg="rosserial_python" type="serial_node.py">
		<param name="port" value="/dev/ttyACM0"/>
		<output="screen"/>
	</node>
		
	<node name="nox_controller" pkg="nox" type="nox_controller">
		<param name="publish_tf" value="true" />
        <param name="publish_rate" value="10.0" />
        <param name="linear_scale_positive" value="1.025" />
        <param name="linear_scale_negative" value="1.025" />
        <param name="angular_scale_positive" value="1.078" />
        <param name="angular_scale_negative" value="1.078" />
		<param name="angular_scale_accel" value="0.0" />
	</node>	
	
				<!--  ************** Sensors *************** 
	<include file="$(find freenect_launch)/launch/freenect.launch" />
	<node name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan">
		<remap from="image" to="camera/depth/image_raw" />
	</node>
	 -->
</launch>
